managed/services/inventory/agents.go

```diff
@@ -158,6 +159,54 @@ func (as *AgentsService) Get(ctx context.Context, id string) (inventorypb.Agent,
 	return res, e
 }
 
+// Logs by Agent ID.
+//nolint:unparam
+func (as *AgentsService) Logs(ctx context.Context, id string) ([]string, error) {
+	agent, err := models.FindAgentByID(as.db.Querier, id)
+	if err != nil {
+		return nil, err
+	}
+
+	var pmmAgentID *string
+
+	switch agent.AgentType {
+	case models.PMMAgentType:
```

Comment:
can we group some of the `case`?

Reply:
i copy-paste from common cases structure and don't want to group, because this version be easy to update

Reply:
Why? it is very hard to read

Reply:
how can this help with future "easy" update if it is a copy-paste

Reply:
Can we extract getting pmmAgentID to some function, I'm pretty sure that we might need it in other places.